src/sagemaker_training/environment.py

```diff
@@ -312,6 +312,7 @@ def num_neurons():  # type: () -> int
         neuron_cores = 0
         for item in j:
             neuron_cores += item.get("nc_count", 0)
+        logger.info(f"Found {neuron_cores} on this instance")
```

Comment:
Should this be`logger.info(f"Found {neuron_cores} Neurons on this instance")`?

Reply:
Thanks for pointing this typo. Updated.